List/list.css

```diff
@@ -0,0 +1,64 @@
+#list{
+    margin-left: 1rem;
+}
+
+.simple-list{
+    background-color: rgb(18, 128, 224);
+    height:fit-content;
+    width: fit-content;
+    padding: 10px;
+    cursor: pointer;
+    color: white;
+    letter-spacing: 1px;
+    border-bottom: 1px solid rgb(187, 184, 184);
+}
+
+.list-style-none{
+    list-style: none;
+}
+
+.simple-list:hover{
+    background-color: rgb(75, 148, 216);
+}
+
+.list-items{
+    cursor: pointer;
+    padding: 10px;
+    column-gap: 16px;
```

Comment:
suggestion: use rem

Reply:
for above 10px i hv used rem. will modify